src/Sulu/Bundle/WebsiteBundle/EventSubscriber/DomainEventEventSubscriber.php

```diff
@@ -53,6 +62,11 @@ public function onCacheClear(CacheClearEvent $event): void
     {
         $tags = $event->getTags();
         if (null === $tags || 0 === \count($tags)) {
+            /** @var Webspace $webspace */
+            foreach ($this->webspaceManager->getWebspaceCollection() as $webspace) {
```

Comment:
looks good to me, but i think the events are not dispatched because there is no flush 🙈

Reply:
think we could use the `DomainEventDispatcher` instead of the `DomainEventCollector` here